### PR TITLE
Handling tree computation timeouts

### LINKED ISSUE #112: specific request timeout errors
The node client should emit a specific error on request timeout

### PR DESCRIPTION
**Pull request by cloderic**
fixes #112

### CODE REVIEW

**Comment by cloderic** [comment]
Had to deactivate the long tests on travis :/

File: test/browser/helper.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import _ from 'lodash';
-import Debug from 'debug';
 import { expect } from 'chai';
+import Debug from 'debug';
```
Comment (Giska) [inline test/browser/helper.js]: I don't think linter is allowing this

File: test/nodejs/helper.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import _ from 'lodash';
+import { expect } from 'chai';
 import Debug from 'debug';
 import dotenv from 'dotenv';
-import { expect } from 'chai';
```
Comment (Giska) [inline test/nodejs/helper.js]: same here

File: test/browser/helper.js (hunk 1 above)
Comment (cloderic) [reply test/browser/helper.js]: it does, otherwise it'd work...

**Pull request merged**